packages/flutter/lib/src/material/paginated_data_table.dart

```diff
@@ -158,6 +160,11 @@ class PaginatedDataTable extends StatefulWidget {
   ///
   /// This value is optional and defaults to 56.0 if not specified.
   final double headingRowHeight;
+  
```

Comment:
There is an invisible trailing space here I think, according to the analyzer failure.